src/server/report/radio.js

```diff
@@ -0,0 +1,63 @@
+import { css, html } from 'lit';
+import { getId } from './common.js';
+
+export const RADIO_STYLE = css`
```

Comment:
> No visual or functional changes here.

Are the radio styles new?  Or am I losing it, where did they come from before?

Reply:
Nvm, I am losing it.  They were previously "pills" lol

Reply:
Haha yeah I took the opportunity to rename them to radio.